app/views/ContactEmailView.scala.html

```diff
@@ -26,9 +26,9 @@
     govukButton: GovukButton
 )
 
-@(form: Form[_], affinityType: AffinityType, mode: Mode)(implicit request: Request[_], messages: Messages)
+@(form: Form[_], affinityType: AffinityType, contactName: String = "", mode: Mode)(implicit request: Request[_], messages: Messages)
```

Comment:
I don't think contactName should be defaulted in the view, especially not to an empty string since we never want that to happen.

Reply:
I think the reason shailesh has done this is because the page is dynamic depending on whether its an individual or organisation. Line 42 - the heading is determined using the affinityType - So we only pass the name into the organisation message key else it would be an empty string... 

Are you suggesting using an option instead? or passing a message key directly from the controller depending on the affinity type?

Reply:
it's just that the default doesn't need to be there, it's never used anyway - it always gets whatever the output from the getContactName method is in the controller and then it will just be ignored if the individual message doesn't take a parameter